testdata/gocloak-realm.json

```diff
@@ -289,6 +289,59 @@
         "offline_access"
       ]
     },
+    {
```

Comment:
Hmmm, it's good, but then the tests will not work on a different environment. I would prefer to see a creating client in the test instead of this.

Reply:
I've moved the client creation into the test as per your feedback